idutils/ext.py

```diff
@@ -51,6 +51,8 @@ def get_scheme_config_func():
 from threading import Lock
 
 from importlib_metadata import entry_points
+# when Python >=3.12, remove importlib_metadata and replace with:
+# from importlib.metadata import entry_points
```

Comment:
minor: I guess we don't want a python check and already import the correct package there?

Reply:
I am hesitating, because this will be copy/pasted around for sure.... not sure...